src/citrine/resources/workflow.py

```diff
@@ -28,9 +31,82 @@ def __init__(self, project_id: UUID, session: Session):
         self.project_id = project_id
         self.session: Session = session
 
+    def list(self,
+             page: Optional[int] = None,
+             per_page: int = 100) -> Iterable[ResourceType]:
```

Comment:
```suggestion
             per_page: int = 100) -> Iterable[Workflow]:
```
We know the concrete type here, right?